.versionbot/CHANGELOG.yml

```diff
@@ -0,0 +1,124 @@
+- commits:
+    "0":
```

Comment:
This is the bug I mentioned on Flowdock, `commits` should be an array but `yq` changed how it handles arrays so we need to update the script :)

Reply:
Got it :)